.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+    // Use IntelliSense to learn about possible Node.js debug attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Mocha Tests",
+            "program": "${workspaceRoot}/node_modules/mocha/bin/_mocha",
+            "args": [
+                "--compilers", "js:babel-core/register",
+                "--require", "babel-polyfill",
+                "${workspaceRoot}/test/**/*.spec.js",
+                "--debug"
+            ]
+        }
+    ]
+}
```

Comment:
No, not required.   It does help however to have the debugging just work when I press F5 if these files are in the working directory.